README.md

```diff
@@ -68,6 +68,105 @@ poetry run flake8
 poetry run isort -rc .
 ```
 
+## How to Test Integration With Slack
+
+After having developed some new feature, or having in hand what you believe is
+a fix for an existing bug, how do you test it out in a real system in order to
+make sure that your changes do all that you hope they do? The answer; bring up
+the application in your own environment and hook it up to Slack!
+
+In order to do this, there are 6 main things you need to accomplish:
```

Comment:
I feel like specifying `6` things here is going to go stale one day when more or less steps are added, and someone is going to forget to update it. Not a big deal, but if we leave it in, I betchu it happens

Reply:
Yeah, I agree.

- [ ] Remove reference to 6 steps.